Loop/Managers/ServicesManager.swift

```diff
@@ -23,13 +23,16 @@ class ServicesManager {
 
     private let servicesLock = UnfairLock()
 
-    init(pluginManager: PluginManager, deviceDataManager: DeviceDataManager) {
+    init(
+        pluginManager: PluginManager,
+        deviceDataManager: DeviceDataManager,
+        remoteDataServicesManager: RemoteDataServicesManager,
```

Comment:
[nit] I'd order the parameters to match the declarations above (i.e. `analyticsServicesManager` then `loggingServicesManager` then `remoteDataServicesManager`).